### PR TITLE
Testcafe : Mutualize class 

### PR DESCRIPTION
**Pull request by CozyKourai**
- [x] mutualize viewer
- [x] split photos-model into model and timeline
- [x] split photo-album into album**s** and album

### CODE REVIEW

**Comment by CozyKourai** [comment]
[(𝐒) Mutualiser le viewer sur drive et photos](https://trello.com/c/Okh2DRJ9/1812-%F0%9D%90%92-mutualiser-le-viewer-sur-drive-et-photos)

**Comment by CozyKourai** [comment]
[(𝐒) Mutualiser le viewer sur drive et photos](https://trello.com/c/Okh2DRJ9/1812-%F0%9D%90%92-mutualiser-le-viewer-sur-drive-et-photos)

**Comment by CozyKourai** [comment]
[(𝐒) Mutualiser le viewer sur drive et photos](https://trello.com/c/Okh2DRJ9/1812-%F0%9D%90%92-mutualiser-le-viewer-sur-drive-et-photos)

**Comment by CozyKourai** [comment]
[(𝐒) Mutualiser le viewer sur drive et photos](https://trello.com/c/Okh2DRJ9/1812-%F0%9D%90%92-mutualiser-le-viewer-sur-drive-et-photos)

File: testcafe/tests/drive/public-viewer-feature.js [hunk 1]
```diff
@@ -151,21 +151,21 @@ test(`${TEST_PUBLIC_VIEWER_ZIP}`, async t => {
...
   )
+
   await publicViewerPage.checkViewerNavigation(
     `${FEATURE_PREFIX}/${TEST_PUBLIC_VIEWER_ZIP}-nav`,
     data.FILE_ZIP,
     3
   )
 
-  await publicViewerPage.checkCommonViewerDownload(data.FILE_ZIP)
+  await publicViewerPage.openFileAndcheckCommonViewerDownload(data.FILE_ZIP)
```
Comment (Crash--) [inline testcafe/tests/drive/public-viewer-feature.js]: AndCheck

File: testcafe/tests/drive/public-viewer-feature.js [hunk 2]
```diff
@@ -225,18 +225,18 @@ test(`${TEST_PUBLIC_VIEWER_AUDIO}`, async t => {
...
   await t.fixtureCtx.vr.setMaksCoordonnates({
-    height: 25,
-    x: 199,
-    width: 90,
-    y: 409
+    height: 27,
+    x: 160,
+    width: 108,
+    y: 415
   })
```
Comment (Crash--) [inline testcafe/tests/drive/public-viewer-feature.js]: is it wanted in this PR ?

File: testcafe/tests/drive/public-viewer-feature.js [hunk 3]
```diff
@@ -301,8 +301,8 @@ test(`${TEST_DELETE_FOLDER}`, async t => {
   console.group(`↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_DELETE_FOLDER}`)
   await privateDrivePage.goToFolder(TEST_CREATE_FOLDER)
   await t.fixtureCtx.vr.setMaksCoordonnates({
-    height: 935,
-    x: 916,
+    height: 916,
+    x: 935,
```
Comment (Crash--) [inline testcafe/tests/drive/public-viewer-feature.js]: same, wanted?

File: testcafe/tests/drive/public-viewer-feature.js (hunk 2 above)
Comment (CozyKourai) [reply testcafe/tests/drive/public-viewer-feature.js]: I saw my masks were a bit off when checking my screenshots, so I modify the mask.
If you prefer, I can remove the change here and add a new PR for fixing the masks

File: testcafe/tests/drive/public-viewer-feature.js (hunk 3 above)
Comment (CozyKourai) [reply testcafe/tests/drive/public-viewer-feature.js]: same thing

File: testcafe/tests/drive/public-viewer-feature.js (hunk 2 above)
Comment (Crash--) [reply testcafe/tests/drive/public-viewer-feature.js]: Yes please. 

One PR on goal

Comment (CozyKourai) [reply testcafe/tests/drive/public-viewer-feature.js]: Done : https://github.com/cozy/cozy-drive/pull/1538

**Pull request merged**